docs/earthly-command/earthly-command.md

```diff
@@ -251,14 +251,10 @@ Also available as an env var setting: `EARTHLY_CI=true`
 In *target mode*, this option is an alias for
 
 ```
---use-inline-cache --save-inline-cache --no-output --strict
+--no-output --strict
 ```
 
-In *artifact* and *image modes* , this option is an alias for
```

Comment:
nit: does `--strict` not apply?

Reply:
I have no idea. The old docs did not mention it, so I just rolled with it

Reply:
strict applies in either case: https://github.com/earthly/earthly/blob/main/cmd/earthly/build_cmd.go#L58